ghost/core/core/frontend/helpers/comments.js

```diff
@@ -38,18 +39,20 @@ async function comments(options) {
     }
 
     const frontendKey = await getFrontendKey();
+    const {scriptUrl, stylesUrl, appVersion} = getFrontendAppConfig('comments');
 
     const data = {
         'ghost-comments': urlUtils.getSiteUrl(),
         api: urlUtils.urlFor('api', {type: 'content'}, true),
         admin: urlUtils.urlFor('admin', true),
         key: frontendKey,
+        styles: stylesUrl,
         'post-id': this.id,
         'sentry-dsn': '', /* todo: insert sentry dsn key here */
         'color-scheme': colorScheme,
         'avatar-saturation': avatarSaturation,
         'accent-color': accentColor,
-        'app-version': config.get('comments:version'),
+        'app-version': appVersion,
         'comments-enabled': commentsEnabled
     };
 
```

Comment:
Can we switch to the new util `getDataAttributes` util below? (sorry, GitHub won't let me comment on those lines 🙃)

Reply:
Done ✅